website/source/layouts/docs.erb

```diff
@@ -552,6 +552,38 @@
           <li<%= sidebar_current("docs-internals-plugins") %>>
             <a href="/docs/internals/internal-plugins.html">Internal Plugins</a>
           </li>
+					<li<%= sidebar_current("docs-internals-provider-guide") %>>
+            <a href="/docs/internals/providers/index.html">Writing Your Own Provider</a>
+            <ul class="nav">
+              <li<%= sidebar_current("docs-internals-provider-guide-lifecycle") %>>
+                <a href="/docs/internals/providers/lifecycle.html">Understanding the Terraform Lifecycle</a>
+              </li>
+              <li<%= sidebar_current("docs-internals-provider-guide-vs-resources") %>>
+                <a href="/docs/internals/providers/providers-v-resources.html">Providers vs. Resources & Data Sources</a>
```

Comment:
I couldn't find a way to shorten this, and looking at the section, it wasn't really... saying much. I've just deleted it and the link in the sidebar.